docs/core/extensions/snippets/channels/channels/Coordinates.cs

```diff
@@ -0,0 +1,11 @@
+﻿/// <summary>
```

Comment:
I'd remove one of the "channels" subfolders and move everything up a level.

Reply:
Done